js/main.js

```diff
@@ -0,0 +1,27 @@
+var $img = document.querySelector('.journal-image');
+var $title = document.querySelector('#title');
+var $photoURL = document.querySelector('#img-url');
+var $notes = document.querySelector('#notes');
+var $form = document.querySelector('.entry-form');
+
+function updatePhoto(event) {
+  $img.setAttribute('src', $photoURL.value);
+}
+
+function saveEntry(event) {
+  event.preventDefault();
+  var newEntry = {};
+  newEntry.title = $title.value;
+  newEntry.photoURL = $photoURL.value;
+  newEntry.notes = $notes.value;
+  newEntry.EntryID = data.nextEntryId;
+  data.nextEntryId++;
+  data.entries.push(newEntry);
+  $img.setAttribute('src', '../images/placeholder-image-square.jpg');
+  $title.value = '';
+  $photoURL.value = '';
+  $notes.value = '';
```

Comment:
You should use the `reset` method on the `form`'s DOM element instead 🤓

Reply:
Oops, I was thinking about the `reload` method I have used the other day and just wanted to avoid an issue like that one 😅 
![image](https://user-images.githubusercontent.com/109567499/189771211-e9109ddf-abd1-439e-b3a5-efc10f5e0379.png)